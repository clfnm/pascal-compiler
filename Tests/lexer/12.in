12abc + a - 3;
nnn
***
888**(])
23.2
22/
/
.,
