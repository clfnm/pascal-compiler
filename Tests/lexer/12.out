1	1	Id	A	A
1	2	Separator	Dot	.
1	3	Eof	EOF	